front/src/components/gun-part-card/GunPartCard.tsx

```diff
@@ -1,49 +1,37 @@
-import { Product } from "../../schema/NFTCatalogSchema";
-import { useStyles } from "./GunPartCardStyles";
-import { GCCard, GCCardSection, GCImage, GCText } from "../../gc-components";
-import { Icon3dCubeSphere } from "@tabler/icons";
+import {Product} from "../../schema/NFTCatalogSchema";
```

Comment:
i dont understand? do you use prettier or no, i think that we should have spaces inside the {} in import, but you dalete them here

Reply:
I have no idea how this works, sometimes it add the spaces and sometimes not
@AbrosimovVladislav HELP